atoms/default/client/css/_thrasher.scss

```diff
@@ -15,11 +15,286 @@
+
+      @include mc(phablet) {
+        padding-left: 8px;
+        position: relative;
+        margin-top: -5px;
+      }
+    }
+
+    &__headline {
+      margin: 0 0 24px;
+      font-size: 28px;
+      line-height: 115%;
+      font-family: $font-headline;
+      font-weight: 700;
+
+      @include mc(desktop) {
+        line-height: 105%;
+      }
+      @include mc(leftCol) {
+        line-height: 115%;
+      }
+    }
+
+    &__headline-special {
+      color: #510443;
+    }
+
+    &__description {
+      margin: 0 0 4px;
+      font-family: $font-text;
+      font-weight: 700;
+      font-size: 15px;
+      line-height: 150%;
+      color: #333;
+
+      &--link {
+        text-decoration: underline;
+        color: inherit;
+
+        &:visited,
+        &:active {
+          color: inherit;
+        }
+      }
+    }
+
+    &__embed-container {
+      position: relative;
+      display: flex;
+      margin-top: auto;
+
+      iframe {
+        max-width: 100%;
+        min-height: 65px;
+        flex: 1;
+        margin-left: unset; // override .email-sub__iframe rule
+
+        @include mc(mobileLandscape) {
+          min-height: 55px;
+        }
+      }
+    }
+
+    &__cta-button {
+      display: none;
+      align-self: flex-start;
+
+      margin-top: auto;
+      height: 36px;
+      width: 118px;
+      box-sizing: border-box;
+      border-radius: 20px;
+      padding: 7px 15px;
+      position: relative;
+      align-items: center;
+      background-color: #052961;
+      color: #fff;
+
+      span {
+        line-height: 1rem;
+        font-size: 17px;
+        font-family: $font-sans;
+        padding: 0;
+        font-weight: 700;
+      }
+
+      svg {
+        position: absolute;
+        right: 9.2px;
+        padding: 0;
+        display: inline-flex;
+        width: 30px;
+        height: auto;
+        justify-content: center;
+        align-items: center;
+        vertical-align: middle;
+        box-sizing: border-box;
+        fill: #fff;
+      }
+    }
+
+    &__privacy {
+      padding: 8px 0 0;
+      font-family: $font-sans;
+      font-weight: 300;
+      font-size: 12px;
+      line-height: 16.2px;
+
+      p {
+        margin: 0 0 0.5em;
+      }
+
+      a,
+      a:any-link {
+        color: inherit;
+        text-decoration: underline;
+      }
+
+      b {
+        font-weight: 400;
+        color: yellow;
+      }
+
+      @include mc(desktop) {
+        padding: 0;
+      }
+    }
+
+    &__frequency-block {
+      display: inline-flex;
+      justify-content: flex-start;
+      align-items: center;
+
+      span {
+        font-family: $font-sans;
+        font-weight: 700;
+        font-size: 14px;
+        margin-left: 3px;
+      }
+    }
+
+    &__newsletters-image {
+      max-width: 170px;
+      width: 70px;
+      height: auto;
+
+      @include mc(desktop) {
+        width: 100%;
+      }
+    }
+    
+    &__thrasher-image {
+      display: none;
+
+      &-mobile {
```

Comment:
This is only a convention, so not important and makes no difference to how the code works, but when using a 'modifier' css class, its usual (in this style of css anyway) to use a double dash before the modifier bit, (so `house-to-home-thrasher__thrasher-image--mobile` rather than `house-to-home-thrasher__thrasher-image-mobile`).

There's no rule that we have to use that pattern, but it does make it easier to know what the classes are for and how they relate to each other- if you're interested the site about the pattern is https://getbem.com/introduction/

Reply:
Modifier changed